lib/types/src/types.rs

```diff
@@ -585,17 +585,14 @@ pub struct FastGasCounter {
     pub burnt_gas: u64,
     /// Hard gas limit for execution
     pub gas_limit: u64,
-    /// Single WASM opcode cost
-    pub opcode_cost: u64,
```

Comment:
I don't think I noticed any changes to vm_offsets as a result of this change. Is it truly not necessary? Note to self: Double check.

Reply:
VMContext stores a pointer to FastGasCounter, so changing the size of this struct should not change anything there. (Yes, there are probably lots of perf gains to be had around gas instrumentation, but that’ll be for later changes)